.changeset/sad-dodos-sell.md

```diff
@@ -0,0 +1,11 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/fluid-static": minor
+---
+---
+"section": deprecation
+---
+
+Marked `IContainerRuntimeOptions.enableGroupedBatching` as deprecated
```

Comment:
```suggestion
IContainerRuntimeOptions.enableGroupedBatching is now deprecated
```

Reply:
Curious if removing the inline code formatting was intentional. I personally do like to see it even in headers.

Reply:
Applied changes